packages/query-sync-storage-persister/src/index.ts

```diff
@@ -42,7 +43,7 @@ export function createSyncStoragePersister({
   deserialize = JSON.parse,
   retry,
 }: CreateSyncStoragePersisterOptions): Persister {
-  if (typeof storage !== 'undefined') {
+  if (storage) {
```

Comment:
```suggestion
  if (storage != null) {
```

This is actually a good place to use `!= null`. It will guard against `null` and `undefined` but be true for other potentially `falsy` values like `0`

Reply:
yeah but we have types for that 😅. It's not allowed to pass in a storage that's a string or a boolean. In those cases, I would prefer to make the check that has the least amount of code :)

Reply:
I'm easy either way - let me know what the preference is and I can change it (or leave it as is).

Reply:
Leave it as-is